source/includes/administration/_invoices.md

```diff
@@ -2136,3 +2136,52 @@ curl  -X GET 'https://cloudmc_endpoint/rest/invoices/download?invoice_id=3f7b7cc
 | Required | &nbsp; |
 | --- | --- |
 | `invoice_id`<br/>*UUID* | The id of the invoice. |
+
+
+<!-------------------- FLAG INVOICE -------------------->
+
+`POST /invoices/:invoiceId/flag`
+
+Flag an invoice with a message to prevent the invoice from being automatically issued to the customer. 
+Once an invoice is flagged, pricing configuration changes can be made. 
+If the configuration changes impact the previously flagged invoice, it will be voided and a new invoice will be generated. 
+Note: the flag will be maintained even after a new invoice is generated. In order to clear the flag the invoice must be approved. 
+
+
+```shell
+# Approve a draft invoice
+curl -X POST "https://cloudmc_endpoint/rest/invoices/20e9b8d8-b1cb-4462-b6e8-fbb8416b2cbb/flag" \
+   -H "MC-Api-Key: your_api_key"
+```
+
+| Required | &nbsp; |
+| --- | --- |
+| `invoiceId`<br/>*UUID* | The id of the invoice to be flagged |
+
```

Comment:
should you also add `message` here?

Reply:
[`c9b3740` (#501)](https://github.com/cloudops/cloudmc-api-docs/pull/501/commits/c9b3740770f3467a99dcb14babb00eb5cc8eda90)